app/frontend/src/features/profile/view/leaveReference/ReferenceForm.tsx

```diff
@@ -0,0 +1,127 @@
+import { Alert } from "@material-ui/lab";
+import {
+  INVALID_STEP,
+  RATING_STEP,
+  REFERENCE_STEP,
+  SUBMIT_STEP,
+} from "features/profile/constants";
+import Appropriate from "features/profile/view/leaveReference/formSteps/Appropriate";
+import Rating from "features/profile/view/leaveReference/formSteps/Rating";
+import SubmitReference from "features/profile/view/leaveReference/formSteps/submit/SubmitReference";
+import Text from "features/profile/view/leaveReference/formSteps/Text";
+import { User } from "pb/api_pb";
+import { ReferenceType } from "pb/references_pb";
+import { useState } from "react";
+import { useRouteMatch } from "react-router-dom";
+import { leaveReferenceBaseRoute, referenceTypeRoute } from "routes";
+import makeStyles from "utils/makeStyles";
+
+export const useReferenceStyles = makeStyles((theme) => ({
+  alert: {
+    marginBottom: theme.spacing(3),
+  },
+  buttonContainer: {
+    display: "flex",
+    justifyContent: "center",
+    paddingTop: theme.spacing(1),
+  },
+  card: {
+    marginTop: theme.spacing(2),
+    marginBottom: theme.spacing(1),
+  },
+  form: {
+    marginBottom: theme.spacing(2),
+  },
+  text: {
+    "& > .MuiInputBase-root": {
+      width: "100%",
+    },
+    marginTop: theme.spacing(1),
+    [theme.breakpoints.up("md")]: {
+      "& > .MuiInputBase-root": {
+        width: 400,
+      },
+    },
+  },
+}));
+
+export type ReferenceContextFormData = {
+  text: string;
+  wasAppropriate: string;
```

Comment:
make this boolean

Reply:
Unfortunately this won't work – when using the boolean values, the value in onChange function of RadioGroup gets transformed to String. See [this issue](https://github.com/mui-org/material-ui/issues/13250).